test/integration.nix

```diff
@@ -78,5 +79,21 @@ pkgs.nixosTest {
         assert h[1] == "/tmp/secrets/passwordfile-user1.age", "filename is incorrect"
         assert len(h[0].strip()) == 64, "hash length is incorrect"
     assert before_hash[0] != after_hash[0], "hash did not change with rekeying"
+
+    # user1 can edit passwordfile-user1.age
+    system1.wait_for_file("/tmp/")
+    system1.send_chars("cd /tmp/secrets; EDITOR=cat agenix -e passwordfile-user1.age\n")
+    system1.send_chars("echo $? >/tmp/exit_code\n")
+    system1.wait_for_file("/tmp/exit_code")
+    assert "0" in system1.succeed("cat /tmp/exit_code")
+    system1.send_chars("rm /tmp/exit_code\n")
```

Comment:
No, because that runs as root, and I was trying to run as user1.

Reply:
Oh, I get it -- also answers my other Q about send_chars.

Because you've gone through the login process above, this is basically like typing those characters into that (now logged in) terminal, as opposed to succeed which runs as root.

Thanks!